index.html

```diff
@@ -909,7 +909,9 @@ <h2>RDF Serialization/Deserialization</h2>
 
   <p><a>JSON-LD Processors</a> MUST NOT
     attempt to correct malformed <a>IRIs</a> or language tags;
-    however, they MAY issue validation warnings. IRIs are not modified other
+    however, they SHOULD issue validation warnings.
+    <span class="changed">Invalid language tags MUST be ignored.</span>
```

Comment:
What does this mean exactly?

For example, in toRdf, should the whole triple be omitted, or should the object become a literal without language tag? (I assume the former?)

I also don't see this mentioned in the algorithms.

Reply:
Probably should just stick with the previous wording, changing MAY to SHOULD. In fact, the algorithms issue a warning and continue with the (possibly invalid) language tag.

In toRdf, such triples would not be emitted, using the provision that only valid triples are emitted, inasmuch as such triples would be invalid.

Reply:
Fixed in 0d4556303057f8712724a510628ee04ba0764694.